tensorflow/python/keras/benchmark/bidirectional_lstm_benchmark_test.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
It might be good to place all benchmarks from keras.io/examples into one file. Can we rename this file as "keras_examples_benchmark_test.py"?

Reply:
> It might be good to place all benchmarks from keras.io/examples into one file. Can we rename this file as "keras_examples_benchmark_test.py"?

Sounds great. We can separate examples by different class names or like [here](https://github.com/tensorflow/tensorflow/blob/a24767dcaeac10dd87b01ac4de27f0f7ff1e3c55/tensorflow/python/keras/benchmark/keras_cpu_benchmark_test.py#L122), we can define the functions.